packages/dbos-cloud/applications/deploy-app-code.ts

```diff
@@ -73,6 +77,10 @@ export async function deployAppCode(host: string, docker: boolean): Promise<numb
           logger.info(`If ${appName} takes too long to become available, check its logs at...`);
```

Comment:
Can we also provide the dbos-cloud CLI command to look up applications logs here?